make/addon.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+#
+# Helper script for addons: deploy and destroy of argocd, fluxcd, tsb-monitoring and external-dns.
+BASE_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+export BASE_DIR
+
+# shellcheck source=/dev/null
+source "${BASE_DIR}/helpers.sh"
+# shellcheck source=/dev/null
+source "${BASE_DIR}/variables.sh"
+
+ACTION=${1}
+CLOUD_PROVIDER=${2}
+ADDON_NAME=${3}
+
+# Validate input values.
+SUPPORTED_ACTIONS=("deploy_infra" "destroy_infra" "refresh_token_k8s")
```

Comment:
thinking out loud without ability to test it, I assume supported_actions will reflect makefile targets?

Reply:
I will align `SUPPORTED_ACTIONS` to match the exact same name as the original `makefile` targets.